fluidefi/fluidefi-caspernet-aggregator-tools
# Block Finder

**Summary of change:**
- Added a new micro-service, block finder, which accepts block number as an argument and returns the block. 

**List of tasks:** 


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-datastore-fetcher`
```bash
git checkout feature/block-hours-datastore-fetcher
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-finder test has passed.
